options/locale/locale_en-US.ini

```diff
@@ -1093,7 +1093,6 @@ editor.cannot_edit_non_text_files = Binary files cannot be edited in the web int
 editor.edit_this_file = Edit File
 editor.this_file_locked = File is locked
 editor.must_be_on_a_branch = You must be on a branch to make or propose changes to this file.
-editor.only_copy_raw = You may only copy raw text files.
```

Comment:
Shouldn't we show a tooltip when a file can't be copied? (e.g. `docx` file)

Reply:
I think disabled state ought to be enough, but I guess we could introduce a "This file type can not be copied" tooltip for additional clarity.

Reply:
Done

<img width="264" alt="Screenshot 2022-11-18 at 19 24 54" src="https://user-images.githubusercontent.com/115237/202781679-a652fbba-53d6-4d46-b440-232576781851.png">